addons/l10n_ar_withholding/wizards/account_payment_register_views.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_payment_register_form" model="ir.ui.view">
+        <field name="name">account.payment.register.form</field>
+        <field name="model">account.payment.register</field>
+        <field name="inherit_id" ref="l10n_latam_check.view_account_payment_register_form"/>
+        <field name="arch" type="xml">
+            <group>
+                <group name="withholdings" colspan="2" invisible="country_code != 'AR'">
+                    <field name="l10n_ar_withholding_ids" invisible="not can_edit_wizard or (can_group_payments and not group_payment)">
+                        <tree editable="bottom">
+                            <field name="withholding_sequence_id" column_invisible="True"/>
+                            <field name="company_id" column_invisible="True"/>
+                            <field name="currency_id" column_invisible="True"/>
+                            <field name="tax_id" domain="[('l10n_ar_withholding_payment_type', '=', parent.partner_type), ('company_id', '=', company_id)]" options="{'no_open': True, 'no_create': True}"/>
+                            <field name="name" readonly="withholding_sequence_id"/>
+                            <field name="base_amount"/>
+                            <field name="amount"/>
+
+                        </tree>
+                    </field>
+                    <group colspan="2" invisible="not can_edit_wizard or (can_group_payments and not group_payment)">
+                        <label for="l10n_ar_net_amount" string="Net Amount" invisible="l10n_latam_check_id"/>
+                        <label for="l10n_ar_net_amount" string="Check amount" invisible="not l10n_latam_check_id"/>
+                        <field name="l10n_ar_net_amount" nolabel="1" />
+                        <field name="l10n_ar_require_adjust_witholding" invisible="True"/>
+                        <p colspan="2" invisible=" not l10n_ar_require_adjust_witholding" class="alert alert-warning" role="alert">
+                            Ajust total amount, or withholdings amount so that the check amount is the correct one.
```

Comment:
My vote is to keep it, it will give the user a hint on what needs to be adjusted, just to help the user.
Is it needed? Not really, as if they try to validate Odoo will complain that the Check Amount does not match. Still, I think will be good to have a message. If too much trouble, then we remove it.
cc' @jco-odoo @jjscarafia @maq-adhoc

Reply:
fixed